email-mailjet/build.gradle

```diff
@@ -0,0 +1,13 @@
+plugins {
+    id("io.micronaut.build.internal.module")
+}
+
+group = "io.micronaut.email"
+
+dependencies {
+    annotationProcessor("io.micronaut:micronaut-validation")
+    implementation("io.micronaut:micronaut-validation")
+    implementation("com.mailjet:mailjet-client:5.2.0")
```

Comment:
The version should be in `gradle.properties` so we can also have it in core's BOM. This comment applies to all modules.